ci/mkpipeline.py

```diff
@@ -634,17 +621,15 @@ def hash(deps: mzbuild.DependencySet) -> str:
                 step["concurrency"] = 1
                 step["concurrency_group"] = f"rust-build-aarch64/{hash(deps)}"
         elif step.get("id") == "build-x86_64":
-            (deps, check) = get_deps(Arch.X86_64)
-            inputs = step.get("inputs") or []
-            if check and not have_paths_changed(inputs):
+            (deps, check) = get_deps(Arch.X86_64, bazel=True)
+            if check:
```

Comment:
@ParkMyCar I'm not sure about this part. Why did we have to check if any path has changed? (Inputs is set to `*`) This means that currently for every PR the Bazel build will be started.

Reply:
Originally I checked if any path had changed because changes to Bazel specific things like `WORKSPACE` weren't being detected because they weren't included in the deps/hashes of mzbuild. But now they are so it should be totally fine to remove these.